### DISCUSSION TITLE
Please someone help me find a rainbow lua script!!

### DISCUSSION THREAD

**Discussion by Rikas2311** [ixjf/MSIRGB#244] [Q&A]
I'm trying to find a rainbow lua script with smooth transitions, but i can't find one.
I tried to modify the Hue Wheel, but since I have no clue how to program it (because I'm not acquainted with programming), I came here to see if anyone has a rainbow's script or could help me getting one.
If someone knows something about this please respond.

**Comment by ixjf** [comment]
Define "smooth".

**Reply by Rikas2311** [reply]
Like the transition between the colors on hue wheel. 

Something like this:
https://www.youtube.com/watch?v=WT6ibs2h6fQ&t=13s

**Reply by ixjf** [reply]
This might be more difficult than it seems. Neither the HSV nor the RGB color spaces are perceptually smooth (distance between two adjacent colours is not the same). I'll see if I can do something at some point.

**Reply by Rikas2311** [reply]
Ok, thanks a lot!
